fix conflicts in docs about unified_logs vs macadmins_unified_logs

roperzh opened the pull request:
As raised by a community member in [Slack](https://osquery.slack.com/archives/C01DXJL16D8/p1672751794862639), this updates our documentation to account for both `unified_log` and `macadmins_unified_log`.

Per my testing, it should also help with the #9158 bug in Fleet's UI.

I have updated the columns of `macadmins_unified_log` according to what's in the [source code](https://github.com/macadmins/osquery-extension/blob/50f94d0d7001c048e0a5564911e46d1952420a80/tables/unifiedlog/unified_log.go#L47-L69), and modified the example to work.

Since I was there I have also updated the osquery version we use to pull the JSON to `5.6.0` and fixed a small bug related to the examples we pull from there. 

. . . . . . .

roperzh commented on website/api/helpers/get-extended-osquery-schema.js:
````diff
@@ -73,6 +73,13 @@ module.exports = {
 
+        // Grab the last example in the array as is usually the most rich example.
+        // TODO: adjust the UI to show all examples.
+        let examples = expandedTableToPush.examples;
+        if (examples.length > 0) {
+          expandedTableToPush.examples = '```\n' + examples[examples.length- 1] + '\n```';
+        }
+
````
I did this so the examples we pull from the osquery repo are rendered as code, attaching screenshots of before and after:

<img width="864" alt="image" src="https://user-images.githubusercontent.com/4419992/211089655-c8aa6525-576d-4500-b9e8-957461c9c84e.png">


<img width="876" alt="image" src="https://user-images.githubusercontent.com/4419992/211089631-1d095d35-aff2-4e9e-8389-be599627ca3e.png">

mikermcneil replied:
@eashaw

mikermcneil replied:
should this be on line 99?  Not sure fofhand, up to you @eashaw

roperzh replied:
good point, probably in both places OR around line 60?

- this conditional branch covers the case when there aren't any overrides
- something on line 99 should cover the case when we have overrides for other fields, but not for the examples

eashaw commented on website/api/helpers/get-extended-osquery-schema.js:
```diff
@@ -73,6 +73,13 @@ module.exports = {
 
+        // Grab the last example in the array as is usually the most rich example.
+        // TODO: adjust the UI to show all examples.
+        let examples = expandedTableToPush.examples;
```
Can you change the name of this variable to be clear about what it is? e.g., `examplesFromOsquerySchema`

roperzh replied:
done!

eashaw reviewed (changes requested):
Thanks, @roperzh! I requested a few changes.

roperzh commented on website/api/helpers/get-extended-osquery-schema.js:
```diff
@@ -73,6 +73,13 @@ module.exports = {
         expandedTableToPush.fleetRepoUrl = 'https://github.com/fleetdm/fleet/new/main/schema/tables/?filename='+encodeURIComponent('/tables/'+expandedTableToPush.name)+'.yml&value='+encodeURIComponent(sampleYamlSchemaForThisTable);
 
+        // Grab the last example in the array as is usually the most rich example.
+        // TODO: adjust the UI to show all examples.
```
As you suggested, it is about the UI aspect, I wasn't sure if only adjusting the button margin is enough/ok.

If you think it is, I can go ahead and make that change, just let me know!

roperzh commented on website/api/helpers/get-extended-osquery-schema.js:
```diff
@@ -73,6 +73,13 @@ module.exports = {
 
         expandedTableToPush.fleetRepoUrl = 'https://github.com/fleetdm/fleet/new/main/schema/tables/?filename='+encodeURIComponent('/tables/'+expandedTableToPush.name)+'.yml&value='+encodeURIComponent(sampleYamlSchemaForThisTable);
 
+        // Grab the last example in the array as is usually the most rich example.
```
thanks! adjusted the text to mention that we only pick one to accommodate for the UI, but I still mention why the one we pick is the last, let me know if it looks ok!

roperzh commented on website/api/helpers/get-extended-osquery-schema.js:
````diff
@@ -73,6 +73,13 @@ module.exports = {
+        // TODO: adjust the UI to show all examples.
+        let examples = expandedTableToPush.examples;
+        if (examples.length > 0) {
+          expandedTableToPush.examples = '```\n' + examples[examples.length- 1] + '\n```';
````
👍 added!

roperzh commented:
@eashaw thanks for the review! I addressed your comments, could you please take a look again? Let me know if we should tackle modifying the styles so we can display multiple examples at once here or separately.

mikermcneil commented:
Thanks @roperzh!

mikermcneil commented:
https://github.com/fleetdm/fleet/pull/9249 makes it so this can be autoapproved
